distribution/ddf-common/src/main/resources/etc/shell.init.script

```diff
@@ -30,11 +30,35 @@ help = { *:help $args | more } ;
 man = { help $args } ;
 log:list = { log:get ALL } ;
 service:get = { $.context getService ($.context getServiceReference $args) };
+
+env = { shell:set $args }
+edit = { shell:nano $args }
```

Comment:
Do we really want to leave this option in, to enter an editing session in the karaf shell?